frontend/src/components/page_content/profile/profile.js

```diff
@@ -87,12 +113,53 @@ class Profile extends React.Component {
       overflow-x: hidden;
       height: calc(90vh - 10.5rem);
     }
+
+    .error {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      background-color: var(--toast-red-transparent);
+      width: 93%;
+      border-radius: 1rem;
+      margin-top: 1rem;
+      margin-bottom: 1rem;
+      padding: 0.5rem;
+    }
+
+    .error-text {
+      font-size: 0.875rem;
+      color: var(--toast-red);
+    }
+
+    .error-icon {
+      cursor: pointer;
```

Comment:
maybe the icon could be centered? im assuming the error message will only be one line though